README.md

```diff
@@ -33,27 +35,30 @@
 ## Quickstart
 
 ### Installation
+
 To install **NestJS tRPC** with your preferred package manager, you can use any of the following commands:
 
 ```shell
 # npm
-npm install trpc-nestjs
+npm install trpc-nestjs zod
```

Comment:
We should also add the `@trpc/server` here since both `zod` and `@trpc/server` are peer dependencies.